web-auth/src/routes/+page.svelte

```diff
@@ -1,7 +1,17 @@
 <script>
   import Auth from "../components/Auth.svelte";
 
-  let cloudClientIDs = import.meta.env.VITE_RILL_CLOUD_AUTH0_CLIENT_IDS;
+  const cloudClientIDs = import.meta.env.VITE_RILL_CLOUD_AUTH0_CLIENT_IDS;
+  const oktaName = import.meta.env.VITE_OKTA_CONNECTION;
+  const pingFedName = import.meta.env.VITE_PINGFED_CONNECTION;
+  const disableForgotPassDomains = import.meta.env
+    .VITE_DISABLE_FORGOT_PASS_DOMAINS;
 </script>
 
-<Auth {cloudClientIDs} configParams="@@config@@" />
+<Auth
+  {cloudClientIDs}
+  {oktaName}
+  {pingFedName}
+  {disableForgotPassDomains}
+  configParams="@@config@@"
```

Comment:
Why is `configParams` passed in as a prop when it's a constant?

Reply:
`configParams` is also a pretty vague name. I'm having a hard time following what this is.